packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx

```diff
@@ -286,6 +294,27 @@ const ChartConfigTabs: FC = () => {
                                     }
                                 />
                             </InputWrapper>
+                            <GridSettings style={{ marginTop: 10 }}>
+                                <Label style={{ marginTop: 5 }}>Sort</Label>
+                                <HTMLSelect
+                                    options={[
+                                        {
+                                            value: 'ascending',
+                                            label: 'Ascending',
+                                        },
+                                        {
+                                            value: 'descending',
+                                            label: 'Descending',
+                                        },
+                                    ]}
+                                    defaultValue="ascending"
```

Comment:
The bug is probably because we set `defaultValue` instead of `value` with the value from the config